### DISCUSSION TITLE
Pull Shark not updating

### DISCUSSION THREAD

**Discussion by mugambow** [community/community#20182] [Other Feature Feedback, Questions, & Ideas]
Hi, I have completed over 300 pull requests in a private repository from my work account but the badges are not updating on my profile. What are the criteria to get one's contribution to private repositories to appear on their profiles without making the repositories public? I have seen some users who have an `inaccessible` note next to the achievement milestone. `Show Achievements on my profile` is set to on.

**Comment by sanrionpc** [comment]
You will need to enable **Include Private contributions on my profile** in settings

**Comment by Tostisto** [comment]
Set this in you profile setting. This will probably work.
![Screenshot 2022-07-10 212513](https://user-images.githubusercontent.com/47717516/178159180-8561df6f-08a8-4053-bd32-74f913311f6d.png)

**Reply by itsrajatrai** [reply]
I had it enabled and still its not working

**Reply by tony19** [reply]
Achievements apparently only counts actions from public repos despite the *Include private contributions in my profile* setting. This seems like a bug to me.

**Reply by 0rzech** [reply]
The achievement does not update for me even with PRs accepted in public repos. Not that it's important or something, but it doesn't work.

**Comment by JosephRivera-ngt** [comment]
Noticed a weird behavior where it only happens when a PR is merged in only one repo under our org. Any insights?

**Comment by falselunatic** [comment]
i am facing the same problem :(

**Comment by mohamedzhioua** [comment]
take a look on your notifications maybe you need to put them all done (clear all your unread notification of merged pull requests)

**Comment by Varunvaru524** [comment]
this particular thing also happen to me a few days back, but I just toggle unhide and hide for a couple of times and it got fixed, it's definitely a bug, and they should fix it
